### PR TITLE
Page Builder > Floating Page

### PR DESCRIPTION
**Pull request by bryantgillespie**
**Design Example**

ScreenShot 2023-07-28 at 15 24 12@2x

### CODE REVIEW

File: components/Block/FloatingPage.vue [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+			</div>
+		</div>
+	</div>
+</template>
+
+<style scoped lang="scss">
+.floating-page {
+	padding-inline: var(--space-8);
+	padding-block: var(--space-8);
+	margin-inline: auto;
```
Comment (rijkvanzanten) [inline components/Block/FloatingPage.vue:66]: Padding and margin should be handled by the context the block is rendered in, not the block itself

File: components/Block/FloatingPage.vue [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+	box-shadow: var(--shadow-lg);
+	border-radius: var(--rounded-2xl);
+
+	> * + * {
+		margin-block-start: var(--space-8);
+	}
+
+	@media (width > 50rem) {
```
Comment (rijkvanzanten) [inline components/Block/FloatingPage.vue:74]: Lets use container queries for this to handle the 50/50 block usage

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: components/Block/FloatingPage.vue (hunk 1 above)
Comment (bryantgillespie) [reply components/Block/FloatingPage.vue:66]: Gotcha - yeah I'll remove the margin here but the padding is for the content inside the 'page'.
Or should I just wrap this in another div?

ScreenShot 2023-07-28 at 21 20 32@2x

File: components/Block/FloatingPage.vue (hunk 2 above)
Comment (bryantgillespie) [reply components/Block/FloatingPage.vue:74]: Right now the data model is setup to only allow this on a full width page block and not within the 50/50.

File: components/Block/FloatingPage.vue (hunk 1 above)
Comment (rijkvanzanten) [reply components/Block/FloatingPage.vue:66]: This particular padding is for the element itself, so this is the right spot 🙂 sorry 'bout the confusion

File: components/Block/FloatingPage.vue (hunk 2 above)
Comment (rijkvanzanten) [reply components/Block/FloatingPage.vue:74]: _right now_ being the key words there 😄 Lets keep things consistent and make sure any block can be rendered in any context

**Pull request closed without merging**